src/api/payment-requests/payment-requests.md

```diff
@@ -713,6 +714,48 @@ Alternatively you can provide an external transaction Id and the Centrapay [Asse
 | 403    | {% break _ REFUND_NOT_SUPPORTED %}          | The asset type does not support refunds.                                                                                                                                                                                                                                    |
 | 403    | {% break _ REFUND_WINDOW_EXCEEDED %}        | The time since the payment exceeds the window of time a payment request can be refunded in.                                                                                                                                                                                 |
 
+<a name="void">
+### Void a Payment Request **EXPERIMENTAL**
+
+Voiding a payment request will cancel the request and trigger any refunds if necessary.
+
+{% reqspec %}
+  POST '/api/payment-requests/{paymentRequestId}/void'
+  auth 'jwt'
+  path_param 'paymentRequestId', 'MhocUmpxxmgdHjr7DgKoKw'
+{% endreqspec %}
+
+{% h4 Example response payload %}
+{% json %}
+{
+  "type": "voidCancel",
+  "status": "voided",
+	"cancellationReason": "CANCELLED_BY_MERCHANT",
+  "value": {
+    "currency": "NZD",
+    "amount": "1000",
+  },
+  "assetType": "centrapay.nzd.main",
+  "paymentRequestId": "MhocUmpxxmgdHjr7DgKoKw",
+  "merchantName": "Centrapay Café",
+  "merchantId": "26d3Cp3rJmbMHnuNJmks2N",
+  "merchantAccountId": "C4QnjXvj8At6SMsEN4LRi9",
+  "merchantConfigId": "5efbe2fb96c08357bb2b9242",
+  "createdAt": "2021-06-08T04:04:27.426Z",
+  "createdBy": "crn::user:0af834c8-1110-11ec-9072-3e22fb52e878",
+  "paymentRequestCreatedBy": "crn::user:0af834c8-1110-11ec-9072-3e22fb52e878",
+  "activityNumber": "2",
+}
+{% endjson %}
+
+{% h4 Error Responses %}
+
+| Status |                Code                |              Description              |
+| :----- | :--------------------------------- | :------------------------------------ |
+| 403    | {% break _ VOID_WINDOW_EXCEEDED %} | 24 hours have passed since createdAt. |
```

Comment:
> 24 hours have passed since createdAt.

Can we explain this is the void window?

Can we refer to the refund endpoint, someone gets this error they're going to want a fix.

Something like...

> The void window is 24 hours. If you need to reverse this transaction you can use our [Refund][] endpoint.